pom.xml

```diff
@@ -146,20 +146,22 @@
                     <artifactId>maven-jar-plugin</artifactId>
                     <version>3.2.0</version>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-failsafe-plugin</artifactId>
-                    <version>2.22.2</version>
-                    <configuration>
-                        <trimStackTrace>false</trimStackTrace>
-                    </configuration>
-                </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.22.2</version>
                     <configuration>
                         <trimStackTrace>false</trimStackTrace>
+                        <systemProperties>
+                            <property>
+                                <name>jdk.internal.httpclient.disableHostnameVerification</name>
```

Comment:
I am a bit confused why don't we leverage programmic (and automatic) way of setting this properties?

Reply:
It was the previous implementation but it relies on the order of loading of the classes since those values are cached in `jdk.internal.net.http.common.Utils`